packages/core/src/RenderingEngine/VideoViewport.ts

```diff
@@ -965,17 +971,27 @@ class VideoViewport extends Viewport implements IVideoViewport {
     this.renderFrame();
   };
 
+  /**
+   * Creates a transform from video index coordinates to canvas coordinates.
+   */
   protected getTransform() {
     const panWorld: Point2 = this.videoCamera.panWorld;
+    const devicePixelRatio = window.devicePixelRatio || 1;
```

Comment:
does these changes mean the SM viewport also suffer from the DPR bug too?

Reply:
Doesn't seem to, probably because we don't actually render the SM viewport itself.